src/resources/views/crud/columns/select.blade.php

```diff
@@ -1,11 +1,18 @@
 {{-- single relationships (1-1, 1-n) --}}
+@php
+    $attributes = $crud->getRelatedEntriesAttributes($entry, $column['entity'], $column['attribute']);
+@endphp
+
 <span>
-    <?php
-        $attributes = $crud->getModelAttributeFromRelation($entry, $column['entity'], $column['attribute']);
-        if (count($attributes)) {
-            echo e(str_limit(strip_tags(implode(', ', $attributes)), array_key_exists('limit', $column) ? $column['limit'] : 40, '[...]'));
-        } else {
-            echo '-';
-        }
-    ?>
+        @if(count($attributes))
+
+        @php($lastKey = array_key_last($attributes))
+        @foreach($attributes as $key => $attribute)
+
+            @php($text = str_limit($attribute, array_key_exists('limit', $column) ? $column['limit'] : 40, '[...]'))
```

Comment:
Any chance we can move most of this logic above, in the PHP section, to leave the ```<span>``` cleaner like in other fields?